File: read_pdf.js
const fs = require('fs');
const pdf = require('pdf-parse');

let dataBuffer = fs.readFileSync('Radip_Shrestha_Resume.pdf');

pdf(dataBuffer).then(function(data) {
    console.log(data.text);
}).catch(err => {
    console.error("Error reading PDF:", err);
});

File: read_pdf.py
import PyPDF2

with open('Radip_Shrestha_Resume.pdf', 'rb') as f:
    reader = PyPDF2.PdfReader(f)
    text = ""
    for page in reader.pages:
        text += page.extract_text() + "\n"

with open('resume.txt', 'w', encoding='utf-8') as f:
    f.write(text)
